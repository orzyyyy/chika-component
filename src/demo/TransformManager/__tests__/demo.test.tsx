import React from 'react';
import { shallow } from 'enzyme';
import 'nino-cli/scripts/setup';
import TransformManagerDemo from '..';

describe('TransformManagerDemo', () => {
  it('render correctly', () => {
    const wrapper = shallow(<TransformManagerDemo />);
    expect(wrapper).toMatchSnapshot();
  });
});
