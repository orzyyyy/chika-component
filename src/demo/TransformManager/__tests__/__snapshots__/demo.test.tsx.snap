// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransformManagerDemo render correctly 1`] = `
<div
  className="TransformManagerDemo"
>
  <Button
    activeStyle={Object {}}
    disabled={false}
    inline={false}
    loading={false}
    onClick={[Function]}
    prefixCls="am-button"
    size="large"
    type="primary"
  >
    next order
  </Button>
  <Button
    activeStyle={Object {}}
    disabled={false}
    inline={false}
    loading={false}
    onClick={[Function]}
    prefixCls="am-button"
    size="large"
  >
    reset
  </Button>
  <Button
    activeStyle={Object {}}
    disabled={false}
    inline={false}
    loading={false}
    onClick={[Function]}
    prefixCls="am-button"
    size="large"
  >
    group2-0
  </Button>
  <TransformManager
    currentGroup="group1"
    currentOrder={0}
  >
    <TransformManagerItem
      className=""
      group="group1"
      key="group1-0"
      order={0}
    >
      group1-0
    </TransformManagerItem>
    <TransformManagerItem
      className=""
      group="group1"
      key="group1-1"
      order={1}
    >
      group1-1
    </TransformManagerItem>
    <TransformManagerItem
      className=""
      group="group1"
      key="group1-2"
      order={2}
    >
      group1-2
    </TransformManagerItem>
    <TransformManagerItem
      className=""
      group="group2"
      key="group2-0"
      order={0}
    >
      group2-0
    </TransformManagerItem>
    <TransformManagerItem
      className=""
      group="group2"
      key="group2-1"
      order={1}
    >
      group2-1
    </TransformManagerItem>
    <TransformManagerItem
      className=""
      group="group2"
      key="group2-2"
      order={2}
    >
      group2-2
    </TransformManagerItem>
  </TransformManager>
</div>
`;
